javascript/javascript2/week3/FastetsPresser/index.js

```diff
@@ -0,0 +1,111 @@
+// Confetti function
+const canvas = document.querySelector("#my-confetti");
+const jsConfetti = new JSConfetti();
+
+let timerStarter;
+let timerStoped = true;
```

Comment:
For the next time you might want to use `timerIsRunning` instead of `timerStopped`. I think it would make the code even more clear :)

Reply:
Agree